packages/embark-compiler/package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "embark-compiler",
+  "version": "4.0.0-beta.0",
+  "author": "Iuri Matias <iuri.matias@gmail.com>",
+  "contributors": [],
+  "description": "Embark is a framework that allows you to easily develop and deploy DApps",
+  "homepage": "https://embark.status.im/",
+  "keywords": [
+    "blockchain",
+    "dapps",
+    "ethereum",
+    "ipfs",
+    "serverless",
+    "solc",
+    "solidity"
+  ],
+  "license": "MIT",
+  "repository": "github:embark-framework/embark",
+  "main": "./dist/index.js",
+  "scripts": {
+    "build": "cross-env BABEL_ENV=node babel src --extensions \".js,.ts\" --out-dir dist --root-mode upward --source-maps",
+    "ci": "npm run qa",
+    "clean": "npx rimraf .nyc_output coverage dist embark-*.tgz package",
+    "lint": "npm-run-all lint:*",
+    "lint:js": "eslint src/",
+    "lint:ts": "tslint -c tslint.json \"src/**/*.ts\"",
+    "package": "npm pack",
+    "qa": "npm-run-all lint typecheck build package",
+    "reset": "npm run clean && npx rimraf node_modules",
+    "start": "npm run watch",
+    "typecheck": "tsc",
+    "watch": "run-p watch:*",
+    "watch:build": "npm run build -- --verbose --watch",
+    "watch:typecheck": "npm run typecheck -- --preserveWatchOutput --watch"
+  },
+  "eslintConfig": {
+    "extends": "../../.eslintrc.json"
+  },
+  "dependencies": {
+    "async": "2.6.1",
```

Comment:
No big deal, but if at some point a dep is added via yarn/npm cli, `package.json` will be rewritten with deps in alpha order. So I'm in the habit of putting them in alpha order so that "surprise" reorders don't happen later.